### PR TITLE
folder name changed to hw-1

### PR DESCRIPTION
**Pull request by msemerni**

### CODE REVIEW

**Comment by msemerni** [comment]
hw-2

File: students/msemerni/hw-1/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        p {
+            margin: 0 0 10px 0;
+        }
+    </style>
+</head>
+<body>
+    <div id="div">
+    </div>
+    <script>
+        let div = document.getElementById("div");
```
Comment (Temu4) [inline students/msemerni/hw-1/index.html:18]: it's better to use `const` instead `let` for the links to DOM elements

File: students/msemerni/hw-1/js/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+let p2 = document.createElement("p");
```
Comment (Temu4) [inline students/msemerni/hw-1/js/index.js:1]: The same

**Review by Temu4 — CHANGES_REQUESTED** [review]

**Comment by Temu4** [comment]
Hey 👋 
You shouldn't add hw-2/hw-3 here.
Please remove them

**Comment by Temu4** [comment]
@msemerni can I help you with something?

**Pull request closed without merging**